src/core/AutomationPattern.cpp

```diff
@@ -72,6 +80,14 @@ AutomationPattern::AutomationPattern( const AutomationPattern & _pat_to_copy ) :
 		m_timeMap[it.key()] = it.value();
 		m_tangents[it.key()] = _pat_to_copy.m_tangents[it.key()];
 	}
+	if( getTrack()->trackContainer()->type() == TrackContainer::BBTrackContainer )
```

Comment:
You are right, it doesn't matter logically at all.

I was thinking more along the lines of Vesa's request to potentially change this behavior in the future.  If the placeholder case is already there, it makes the future edits much quicker IMO.